pkg/operator/k8sutil/prometheus.go

```diff
@@ -44,18 +40,37 @@ func getMonitoringClient() (*monitoringclient.Clientset, error) {
 	return client, nil
 }
 
-// GetServiceMonitor returns servicemonitor or an error
-func GetServiceMonitor(filePath string) (*monitoringv1.ServiceMonitor, error) {
-	file, err := os.ReadFile(filepath.Clean(filePath))
-	if err != nil {
-		return nil, fmt.Errorf("servicemonitor file could not be fetched. %v", err)
-	}
-	var servicemonitor monitoringv1.ServiceMonitor
-	err = k8sYAML.NewYAMLOrJSONDecoder(bytes.NewBufferString(string(file)), 1000).Decode(&servicemonitor)
-	if err != nil {
-		return nil, fmt.Errorf("servicemonitor could not be decoded. %v", err)
+// GetServiceMonitor creates serviceMonitor object template
+func GetServiceMonitor(name string, namespace string) *monitoringv1.ServiceMonitor {
+	return &monitoringv1.ServiceMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels: map[string]string{
+				"team": "rook",
+			},
+		},
+		Spec: monitoringv1.ServiceMonitorSpec{
+			NamespaceSelector: monitoringv1.NamespaceSelector{
+				MatchNames: []string{
+					namespace,
+				},
+			},
+			Selector: metav1.LabelSelector{
+				MatchLabels: map[string]string{
```

Comment:
`mgr_role: active` Is one of the match label

Reply:
@parth-gr We just removed that in #12269 a few days ago, so it's not needed